quarkus-build-caching-extension/src/it/postbuild.groovy

```diff
@@ -0,0 +1,38 @@
+// Utility functions
+String getContent(String fileName) {
+    File f = new File(basedir, fileName)
+    assert f.exists()
+    return f.text
+}
+
+void checkBuildCacheDisabled(String logFile) {
```

Comment:
This is a nit but I would name all these methods in this file with `ensure` or `assert` because once this method passes we know the build cache is disabled. `check` implies a return value.
```suggestion
void ensureBuildCacheDisabled(String logFile) {
```